### PR TITLE
readme update

### PR DESCRIPTION
**Pull request by rhagarty**

### CODE REVIEW

File: data/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+RUN apt-get install -y python-scipy
+RUN pip install scikit-learn
+RUN apt-get install -y libxml2-dev libxmlsec1-dev
+RUN pip install dragnet
+
+#add startupscripts
+RUN apt-get install -y supervisor
+WORKDIR /etc/skel
+
+ADD cels.csv /etc/skel/
```
Comment (stevemart) [inline data/Dockerfile]: do these need to be prefixed with ``data/`` since we moved them?

  ``data/cels.csv``?

File: data/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ADD n8wh.csv /etc/skel/
+ADD qcln.csv /etc/skel/
+ADD watson.csv /etc/skel/
+ADD watson-results.csv /etc/skel/
+ADD watson-results-test.csv /etc/skel/
+ADD binary-classification.png /etc/skel/
+ADD feed-forward.png /etc/skel/
+ADD IBM-Google-Demo-Env.png /etc/skel/
+ADD NN-Results.png /etc/skel/
+ADD Clean_Energy_Watson_V1.0.ipynb /etc/skel/
```
Comment (stevemart) [inline data/Dockerfile]: prefix with `notebooks/` ?

File: data/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+Add CTIUS.csv /etc/skel/
+ADD DAX-SP-FTSE-index.csv /etc/skel/
+ADD eui.csv /etc/skel/
+ADD icln.csv /etc/skel/
+ADD n8wh.csv /etc/skel/
+ADD qcln.csv /etc/skel/
+ADD watson.csv /etc/skel/
+ADD watson-results.csv /etc/skel/
+ADD watson-results-test.csv /etc/skel/
+ADD binary-classification.png /etc/skel/
```
Comment (stevemart) [inline data/Dockerfile]: not sure if adding .pngs adds value, are they used in the notebook?

File: data/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,46 @@
+#must be built on a IBM Power8 system
```
Comment (stevemart) [inline data/Dockerfile]: hmm, maybe have this as a top level resource?

**Comment by stevemart** [comment]
just a few comments about the dockerfile, its fine that this is merged, i like the readme update.

File: data/Dockerfile (hunk 4 above)
Comment (rhagarty) [inline data/Dockerfile]: not sure... just wanted to declare this somewhere. Maybe in the developing doc?

File: data/Dockerfile (hunk 3 above)
Comment (rhagarty) [inline data/Dockerfile]: yes, they are displayed in the notebook

File: data/Dockerfile (hunk 1 above)
Comment (rhagarty) [inline data/Dockerfile]: just added the dockerfile as-is as starting point. will look into this today.

File: data/Dockerfile (hunk 2 above)
Comment (rhagarty) [inline data/Dockerfile]: will investigate

**Pull request merged**